### PR TITLE
Formatting data in coldata class

### PR DESCRIPTION
**Pull request by krlmlr**
The colformat class has been renamed to coldata, the new colformat class aggregates coldata. We'll add title and data type later. This is in preparation for #4, please review the design.

### CODE REVIEW

File: R/coldata.R [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+  new_column(x, title = title, width = width, align = attr(x, "align"))
+}
+
+#' @export
+print.coldata <- function(x, title = "title", ...) {
+  print(format(x, title = title, ...))
+}
+
+new_coldata <- function(x, width = max(crayon::col_nchar(x)), align = "left") {
```
Comment (hadley) [inline R/coldata.R]: We need to export this so people who want to write their own methods can.

File: R/colformat.R [hunk 2]
```diff
@@ -31,96 +31,19 @@
...
-  new_column(x, title = title, width = width, align = attr(x, "align"))
-}
-
-#' @export
-print.colformat <- function(x, title = "title", ...) {
-  print(format(x, title = title, ...))
-}
-
-new_colformat <- function(x, width = max(crayon::col_nchar(x)), align = "left") {
+  data <- coldata(x, ...)
```
Comment (hadley) [inline R/colformat.R]: This is just a place holder, right?

Comment (krlmlr) [inline R/colformat.R]: Yes, there's more to come.

**Comment by krlmlr** [comment]
camelCase or snake_case?

Yes, the intention is to have multiple classes, one for title, one for type, one for body. Later perhaps for the footer if necessary.

**Comment by hadley** [comment]
Do you even need to ask? 😉  🐍

**Pull request merged**